### ISSUE TITLE
It adds extra space inside class attribute when using @error directive

### ISSUE THREAD

**Issue by imre-iglu** [Stillat/blade-parser-typescript#103]
I would first like to thank plugin author for creating this amazing prettier package. It's best one I have used.

I saw that it adds two spaces after the `@error()` directive, when using inside the class attribute.

It turns this HTML:

```blade
<input type="text" class="my-class @error('form.title') has-error @enderror" />
```

Into this:

```blade
<input type="text" class="my-class @error('form.title')  has-error @enderror"/>
````

My `.blade.format.json` looks like this:

```json
{
    "useLaravelPint": false,
    "spacesAfterDirective": 0,
    "spacesAfterControlDirective": 0,
    "phpOptions": {
        "singleQuote": true,
        "phpVersion": "8.1"
    },
    "attributeJsOptions": {
        "semi": true,
        "printWidth": 120
    }
}
```

**Comment by JohnathonKoster** [comment]
Thanks for the report! I'll take a look this upcoming weekend

**Comment by JohnathonKoster** [comment]
Fix released in v2.1.14
